sources/platform/api_v2/api_v2_reference.apib

```diff
@@ -902,6 +902,25 @@ Only builds that are starting or running are aborted. For builds with status `FI
     + Attributes
         - data (BuildAborted, required)
 
+## Build logs [/v2/acts/{actorId}/builds/{buildId}/log{?token,stream,download}]
+
+Check out [Logs](#reference/logs) for full reference.
+
++ Parameters
+
+    + buildId: `HG7ML7M8z78YcAPEB` (string, required) - ID of the actor build.
+    + token: `soSkq9ekdmfOslopH` (string, required) - API authentication token.
+    + stream: false (boolean, required) - If `true` or `1` then the logs will be streamed as long as the run or build is running.
+    + download: false (boolean, required) - If `true` or `1` then the web browser will download the log file rather than open it in a tab.
+
+### Get log [GET]
+
++ Response 200 (text/plain)
+
+2017-07-14T06:00:49.733Z Application started.
```

Comment:
It looks that your changes reporting some semantic issues
![image](https://github.com/apify/apify-docs/assets/11837643/39ff9e81-7480-402e-b00b-0f040c49ff16)